lib/stackprof-webnav/presenter.rb

```diff
@@ -45,7 +45,7 @@ def method_info name
             :callers => callers(frame, info),
             :callees => callees(frame, info),
             :location => file,
-            :source => BetterErrors::CodeFormatter::HTML.new(file, line).output
+            :source => file && line ? BetterErrors::CodeFormatter::HTML.new(file, line).output : nil
```

Comment:
interesting in what scenario these are missing? maybe reproduce in unit test?

Reply:
I could for sure, they're missing in C methods.